front/src/modules/people/components/PeopleCard.tsx

```diff
@@ -72,6 +164,27 @@ export function PeopleCard({
         <StyledTitle>{person.displayName}</StyledTitle>
         {person.jobTitle && <StyledJobTitle>{person.jobTitle}</StyledJobTitle>}
       </StyledCardInfo>
+      {isHovered && (
+        <div ref={refs.setReference}>
+          <StyledIconButton
+            onClick={handleToggleOptions}
+            variant="shadow"
+            size="small"
+            icon={<IconDotsVertical />}
```

Comment:
we need to pass the IconSize (theme.icon.size.md I think). here the 3 dots are too big